Sources/Html/Elements.swift

```diff
@@ -582,16 +583,89 @@ public func meta(keywords: [String]) -> ChildOf<Element.Head> {
   return meta([name(.keywords), content(keywords)])
 }
 
+public func meta(name: String, content: String) -> ChildOf<Element.Head> {
+  return meta([attribute("name", name), Html.content(content)])
+}
+
 public func meta(property: String, content: String) -> ChildOf<Element.Head> {
   return meta([attribute("property", property), Html.content(content)])
 }
 
-public func meta(name: String, content: String) -> ChildOf<Element.Head> {
-  return meta([attribute("name", name), Html.content(content)])
+public enum Viewport {
+  case height(ViewportHeight)
+  case initialScale(Double)
+  case maximumScale(Double)
+  case minimumScale(Double)
+  case userScalable(Bool)
+  case width(ViewportWidth)
+}
+extension Viewport: CustomStringConvertible {
+  public var description: String {
+    switch self {
+    case let .height(px):
+      return "height=\(px.description)"
+    case let .initialScale(scale):
+      return "initial-scale=\(scale)"
+    case let .maximumScale(scale):
+      return "maximum-scale=\(scale)"
+    case let .minimumScale(scale):
+      return "minimum-scale=\(scale)"
+    case let .userScalable(isUserScalable):
+      return "user-scalable=\(isUserScalable ? "yes" : "no")"
+    case let .width(px):
+      return "width=\(px.description)"
+    }
+  }
+}
+public enum ViewportHeight {
+  case deviceHeight
+  case px(Int)
+}
+extension ViewportHeight: CustomStringConvertible {
+  public var description: String {
+    switch self {
+    case .deviceHeight:
+      return "device-height"
+    case let .px(px):
+      return "\(px)"
+    }
+  }
+}
+extension ViewportHeight: ExpressibleByIntegerLiteral {
+  public init(integerLiteral value: Int) {
+    self = .px(value)
+  }
+}
+public enum ViewportWidth {
+  case deviceWidth
+  case px(Int)
+}
+extension ViewportWidth: CustomStringConvertible {
+  public var description: String {
+    switch self {
+    case .deviceWidth:
+      return "device-width"
+    case let .px(px):
+      return "\(px)"
+    }
+  }
+}
+extension ViewportWidth: ExpressibleByIntegerLiteral {
+  public init(integerLiteral value: Int) {
+    self = .px(value)
+  }
+}
+
+public func meta(viewport props: NonEmptyArray<Viewport>) -> ChildOf<Element.Head> {
+  return meta([name(.viewport), content(Array(props).map(get(\.description)).joined(separator: ","))])
+}
+
+public func meta(viewport prop: Viewport, _ props: Viewport...) -> ChildOf<Element.Head> {
```

Comment:
Been doing a thing where I overload functions that take `NonEmpty` with a friendlier forced variadic version. Not sure if it scales well, but it made `Gen` a bit more wieldy.